.buildkite/scripts/lifecycle/post_command.sh

```diff
@@ -25,6 +24,13 @@ if [[ "$IS_TEST_EXECUTION_STEP" == "true" ]]; then
   buildkite-agent artifact upload 'x-pack/test/functional/failure_debug/html/*.html'
   buildkite-agent artifact upload '.es/**/*.hprof'
 
+  if (env.CODE_COVERAGE) {
+    sh 'tar -czf kibana-coverage.tar.gz target/kibana-coverage/**/*'
+    buildkite-agent artifact upload 'kibana-coverage.tar.gz'
```

Comment:
there will be 'kibana-coverage.tar.gz' file for each CI group with functional tests, later we download it and merge in a single report